tests/profile/hmmer/CPtrStore.csv

```diff
@@ -0,0 +1,255 @@
+DigitizeAlignment_call5	DigitizeAlignment_arrayidx
```

Comment:
Umm... good question. They sure don't look like csv, you're right. I'm reusing an existing datalog program and input from another test, as something to generate a log from, so this is borrowed from there, and I'm not sure why it's csv. I can fix these ones, but the problem definitely exists in at least one other place, and probably several.